src/fhi/blocks/_fhi-form-check-tile.block.scss

```diff
@@ -0,0 +1,53 @@
+.fhi-form-check-tile {
+    margin-bottom: $fhi-core-space-3;
+    max-width: 25rem;
```

Comment:
@proand - Not sure about this. Should we allow full width?

Reply:
Hmm... full width feels like the most systematic aproach to me, even though we are pushing som extra work onto the consumer in makin grids etc... I'm 70/30 in favor of full widht ;)

Reply:
Full width it is